run_infersharp.sh

```diff
@@ -15,15 +15,12 @@ rm -r infer-out
 coreLibraryPath=/app/Cilsil/System.Private.CoreLib.dll
 echo $coreLibraryPath
 echo "Copy binaries to a staging folder..."      
-sudo cp $coreLibraryPath $1     
+sudo cp $coreLibraryPath $1
 
 # Run InferSharp analysis.
 sudo dotnet /app/Cilsil/Cilsil.dll translate $1 --outcfg $1/cfg.json --outtenv $1/tenv.json --cfgtxt $1/cfg.txt
+echo -e "\e[1;33mYou may see 'Unable to parse instruction xxx' above. This is expected as we have not yet translated all the CIL instructions, which follow a long tail distribution. We are continuing to expand our .NET translation coverage. \e[0m\n"
+echo -e "Translation completed. Analyzing...\n"
 sudo infer capture 
 sudo mkdir infer-out/captured 
-sudo infer analyzejson --debug --cfg-json $1/cfg.json --tenv-json $1/tenv.json
-reportonfiles=$(echo $3 | sed 's/ /,/g')
-sudo dotnet /app/AnalysisResultParser/AnalysisResultParser.dll infer-out/bugs.txt infer-out/filtered_bugs.txt $reportonfiles
-mkdir -p $2
-sudo cp infer-out/filtered_bugs.txt $2
-echo "Bug report is ready at $2/"
```

Comment:
Should at least keep this message as `echo "Bug report is ready at $1/"` if we don't have a <output_path> input.

Reply:
Infer actually spits out relevant info like this:
```
Found 10 issues (console output truncated to 5, see '/app/infer-out/report.txt' for the full list)
```